sharktank_models/README.md

```diff
@@ -97,3 +91,13 @@ built as part of the [shark-ai project](https://github.com/nod-ai/shark-ai).
 
     See also
     https://docs.pytest.org/en/latest/how-to/output.html#creating-junitxml-format-files
+
+## Running threshold tests
+
+Please refer to [this file](regression_tests/README.md) to run threshold tests
+
+## Running benchmark tests
+
+Please refer to [this file](benchmarks/README.md) to run benchmark tests
```

Comment:
nit: when writing documentation, prefer to put a short, descriptive phrase as the link text, not phrases like "this document": https://developers.google.com/style/link-text. Here I would insert the document titles like `Please refer to [Regression tests](...)`

also... "threshold tests" doesn't carry much meaning to me. I would typically expect "correctness tests" "quality tests" "accuracy tests" or a similar term here in contrast to "benchmark tests"